server/util/jurisdiction_bulk_update.py

```diff
@@ -43,7 +43,11 @@ def bulk_update_jurisdictions(
     with session.begin_nested():
         # Clear existing admins.
         session.query(JurisdictionAdministration).filter(
-            Jurisdiction.election == election
+            JurisdictionAdministration.jurisdiction_id.in_(
+                Jurisdiction.query.filter(Jurisdiction.election == election).values(
```

Comment:
maybe just `Jurisdiction.query.filter_by(election_id=election.id).values(Jurisdiction.id).subquery()` - that's our usual convention for filtering (so it's sure to work) and maybe the subquery makes it more efficient?